lib/rules/custom-property-no-missing-var-function/index.mjs

```diff
@@ -62,29 +62,45 @@ const rule = (primary) => {
 
 			if (IGNORED_PROPERTIES.has(prop.toLowerCase())) return;
 
-			valueParser(value).walk((node) => {
-				if (isVarFunction(node)) return false;
+			valueParser(value).nodes.forEach((childNode) => {
+				check(childNode, decl);
+			});
+		});
+
+		/**
+		 * @param {import('postcss-value-parser').Node} node
```

Comment:
nit

Maybe share the same typedef for this line and l108
i.e. `/** @typedef {import('postcss-value-parser').Node} Node  */`

```suggestion
		 * @param {Node} node
```

Reply:
Fixed it!
[chore: unify types](https://github.com/stylelint/stylelint/pull/8116/commits/c59d67c56dc847a58f661281563cfa2d5ae2c9af)